tests/integration_tests/test_integration.py

```diff
@@ -27,14 +25,12 @@ def test_import() -> None:
 def test_module_attributes() -> None:
     import optuna
 
-    assert hasattr(optuna.integration, "chainermn")
```

Comment:
[question] Do you think we should remove these lines even though we leave the `optuna.integration.chainermn` module for backward compatibility. Or is it possible to install optuna-integration in `tests-integration.yml` and pass these tests?

Reply:
If we continue to conduct this test on the Optuna side, there will continue to be a dependency on Chainer, which is not desirable from the initial motivation of migrating the integration module. Therefore, I think the current approach of completely removing the dependency on Chainer from the test is appropriate. What do you think? @contramundum53 @gen740

Reply:
Does it?
I think we could check whether `"chainermn"` exists without actually importing it.

Reply:
@gen740 pointed out that simply calling `hasattr` on `"chainermn"` would cause it to be imported. I think removing the test is Okay.